<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtistsAlbumsWidget</class>
 <widget class="QWidget" name="ArtistsAlbumsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="mainTopWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tracks in Library</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="initialList">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tracksPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="topWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="backButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">...</string>
              </property>
              <property name="icon">
               <iconset theme="go-previous">
                <normaloff>.</normaloff>.</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="tracksTitle">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Tracks in Library</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="auxiliaryDataLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="buttonWidget" native="true">
            <layout class="QHBoxLayout" name="_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QPushButton" name="enqueueAllButton">
               <property name="text">
                <string>Enqueue All</string>
               </property>
               <property name="icon">
                <iconset theme="view-media-playlist">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="playAllButton">
               <property name="text">
                <string>Play All</string>
               </property>
               <property name="icon">
                <iconset theme="media-playback-start">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="shuffleButton">
               <property name="text">
                <string>Shufle All</string>
               </property>
               <property name="icon">
                <iconset theme="media-playlist-shuffle">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="burnButton">
               <property name="text">
                <string>Burn</string>
               </property>
               <property name="icon">
                <iconset theme="tools-media-optical-burn">
                 <normaloff>.</normaloff>.</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="LibraryListView" name="tracksList">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>LibraryListView</class>
   <extends>QListView</extends>
   <header>library/librarylistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
